docs/examples/DataSet/Working-With-Pandas-and-XArray.ipynb

```diff
@@ -11,7 +11,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "This notebook demonstrates how Pandas and XArray can be used to work with the QCoDeS Dataset. It is not meant as a general introduction to Pandas and XArray. We refer to the official documentation for [Pandas](https://pandas.pydata.org/) and [XArray](http://xarray.pydata.org/en/stable/) for this. This notebook requires that both Pandas and XArray are installed."
+    "This notebook demonstrates how Pandas and XArray can be used to work with the [QCoDeS ``DataSet``](DataSet-class-walkthrough.ipynb). It is not meant as a general introduction to Pandas and XArray. We refer to the official documentation for [Pandas](https://pandas.pydata.org/) and [XArray](http://xarray.pydata.org/en/stable/) for this. This notebook requires that both Pandas and XArray are installed."
```

Comment:
This link does not resolve correctly in the rendered html. I suggest simply removing the ticks

Reply:
yes